static-site/.eslintrc.yaml

```diff
@@ -24,7 +24,13 @@ settings:
     version: "16"
 
 ignorePatterns:
-  - public/
+  - "/*"
+  - "!/app"
+  - "!/components"
+  - "!/pages"
+  - "!/src"
+  - "!/types.d.ts"
+  - "!/vendored"
```

Comment:
Does this intentionally omit things like `data/BaseConfig.js` and `data/SiteConfig.jsx`? I'd think we'd want to keep linting those.

Reply:
Good point, done: 1526d25a974f9133b50e1035d666a49b853ba2bf